rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/local/JobExecutorLocal.groovy

```diff
@@ -59,8 +59,18 @@ class JobExecutorLocal implements JobExecutor {
       new Action0() {
         @Override
         public void call() {
-          ByteArrayOutputStream stdOutAndErr = new ByteArrayOutputStream()
-          PumpStreamHandler pumpStreamHandler = new PumpStreamHandler(stdOutAndErr)
+          PumpStreamHandler pumpStreamHandler
+          ByteArrayOutputStream stdOut
+          ByteArrayOutputStream stdErr
+          if (jobRequest.combineStdOutAndErr) {
+            stdOut = new ByteArrayOutputStream()
+            stdErr = stdOut
```

Comment:
I'm wondering if it would be clearer to just have `stdErr` be null in this case rather than have two references to the same stream.  The check `stdOut.is(stdErr)` below would then just be `stdErr == null` which also seems to convey the intention better.

Reply:
Sure, that seems reasonable.